.github/workflows/e2e-gaia-current-release.yaml

```diff
@@ -1,4 +1,4 @@
-name: End to End testing
+name: End to End testing (Current Gaia)
```

Comment:
Can we put the actual Gaia version here?

Reply:
Done